poetry/templates/archive.html

```diff
@@ -0,0 +1,23 @@
+<!-- wp:template-part {"slug":"header","area":"header"} /-->
+
+<!-- wp:group {"layout":{"type":"constrained"}} -->
+<div class="wp-block-group"><!-- wp:query-title {"type":"archive","style":{"spacing":{"padding":{"bottom":"var:preset|spacing|40"}}}} /-->
+
+<!-- wp:query {"queryId":32,"query":{"perPage":10,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true},"layout":{"type":"default"}} -->
```

Comment:
We don't want the Id on query loops (explained [here](https://github.com/WordPress/community-themes?tab=readme-ov-file#pattern-creation-guidelines) under _Other tips_).